ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -341,6 +341,16 @@ objects:
           ${SSH_PUB_KEY}
         EOF
 
+        # Create a ramdisk for the etcd storage. This helps with disk latency
+        # unpredictability in the OpenStack cloud used by the CI:
+        TF_LOG=debug openshift-install --dir=/tmp/artifacts/installer create ignition-configs --log-level=debug
+        python -c \
+            'import json, sys; j = json.load(sys.stdin); j[u"systemd"][u"units"] = [{u"contents": "[Unit]\nDescription=Mount etcd as a ramdisk\nBefore=local-fs.target\n[Mount]\n What=none\nWhere=/var/lib/etcd\nType=tmpfs\nOptions=size=2G\n[Install]\nWantedBy=local-fs.target", u"enabled": True, u"name":u"var-lib-etcd.mount"}]; json.dump(j, sys.stdout)' \
```

Comment:
With the ramdisk using 2GB of RAM, shouldn't we also bump the RAM for control plane nodes by 2GB as well?

Reply:
Yeah I've been thinking about that. As far as I have seen, there's plenty of free RAM available on the master nodes so it should be okay.

But if you want, I can do a deployment, run the tests and check the memory utilisation over time. There may even be a Prometheus metric that does that.

Reply:
Monitoring data show that the master and worker nodes have plenty of free RAM. The cluster wasn't under load (the tests didn't run) so that's still something we'll need to monitor once this patch merges.

Reply:
Checking this (the Prometheus metrics for this run) it seems the masters have enough memory to handle this: 

https://xdhhjhzz-promecieus.svc.ci.openshift.org/graph?g0.range_input=30m&g0.end_input=2019-10-16%2009%3A52&g0.expr=histogram_quantile(0.99%2C%20rate(etcd_disk_backend_commit_duration_seconds_bucket%5B5m%5D))&g0.tab=0&g1.range_input=1h&g1.expr=etcd_server_health_failures&g1.tab=0&g2.range_input=2h&g2.expr=instance%3Anode_memory_utilisation%3Aratio&g2.tab=0&g3.range_input=1h&g3.expr=node_memory_MemTotal_bytes&g3.tab=0&g4.range_input=1h&g4.expr=node_memory_MemFree_bytes&g4.tab=0